packages/drivers/debugger/api-extractor/api-extractor.legacy.json

```diff
@@ -1,4 +1,8 @@
 {
 	"$schema": "https://developer.microsoft.com/json-schemas/api-extractor/v7/api-extractor.schema.json",
```

Comment:
We also still need to fix the non-legacy report configuration(s) to ensure that our legacy items don't appear in them.